gef.py

```diff
@@ -682,7 +682,7 @@ def is_executable(self) -> bool:
     @property
     def size(self) -> int:
         if self.page_end is None or self.page_start is None:
-            return -1
+            raise AttributeError
```

Comment:
tests are passing and it seemed be fine. plus size of `-1` doesn't mean anything

Reply:
Yeah I think raising an error is better anyway, just don't want it to have some unhandled path. It should actually be impossible to create a section with either set to `None` because in `__init__` we do `kwargs.get("page_start", 0)`, and same for end. We could maybe even drop this check. If it fucks up we will get an exception anyway, but it's breaking an invariant.